packages/api/src/controllers/logController.js

```diff
@@ -115,7 +115,10 @@ const logController = {
       const { name } = req.body;
       const trx = await transaction.start(Model.knex());
       try {
-        const check = await HarvestUseTypeModel.query().where({ farm_id, harvest_use_type_name: name }).first();
+        const check = await HarvestUseTypeModel.query()
+          .where(builder => builder.where('farm_id', farm_id).orWhere('farm_id', null))
```

Comment:
what is the builder for? how is this different from simply calling `.where('farm_id', farm_id)`?

Reply:
want to get the default harvest uses too

Reply:
farm_id is always not null in the request

Reply:
isn't that covered by `.orWhere('farm_id', null)`? or am i mistaken

Reply:
just wondering, aren't default harvest uses covered by `.ofWhere('farm_id', null)`?

Reply:
what's ofWhere?

Reply:
oeWhere***

Reply:
Yes